src/components/PressableWithSecondaryInteraction/pressableWithSecondaryInteractionPropTypes.js

```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
```

Comment:
NAB: Just curious why you chose to declare the prop types in their own file.

Reply:
Just to DRY things up as they are the same in both native and web versions of the `PressableWithSecondaryInteraction`